docs+types: reference/build fixups and unify last_modified type (#326)
DOI-USGS/dataretrieval-python: README.md

print(f"Retrieved {len(df)} continuous gage height measurements")
```

Rating curves come back as a `dict` of rating tables (one per rating file),
keyed by rating id — unlike the other getters, which return a
`(DataFrame, metadata)` pair:

```python
# Get stage-discharge rating curves for a streamgage
ratings = waterdata.get_ratings(monitoring_location_id='USGS-01646500')

for rating_id, table in ratings.items():
    print(f"{rating_id}: {len(table)} rating points")
```

> 

Author: ```suggestion
```

logging.basicConfig(level=logging.DEBUG)
```

### National Ground-Water Monitoring Network (NGWMN)

Access groundwater data aggregated from many state, federal, and local
agencies. NGWMN is a sibling of the Water Data API built on the same engine,

> so chunking, pagination, and result shaping behave the same way:

Author: ```suggestion
agencies. NGWMN uses the same OGC engine as the Water Data API,
so chunking and pagination behave the same way:
```

```python
from dataretrieval import ngwmn

# Find the groundwater monitoring sites in a state
# (state accepts a full name, a postal code like 'WI', or a FIPS code like '55')
sites, metadata = ngwmn.get_sites(state='Wisconsin')

print(f"Found {len(sites)} NGWMN sites in Wisconsin")

# Pull water levels for those sites over a time window. The [:20] keeps this
# example small; drop it to pull the whole state — a multi-site request is
# split into URL-safe chunks automatically.
site_ids = sites['monitoring_location_id'][:20]
water_levels, metadata = ngwmn.get_water_level(
    monitoring_location_id=site_ids,
    datetime=['2022-01-01', '2024-01-01']

> )

Author: ```suggestion
# Pull water levels from the first twenty sites over a time window. 
water_levels, metadata = ngwmn.get_water_level(
    monitoring_location_id=sites['monitoring_location_id'][:20],
    datetime=['2022-01-01', '2024-01-01']
)
```

## Available Data Services

### Modern USGS Water Data APIs (Recommended) — `dataretrieval.waterdata`
- `get_daily`: Daily statistical summaries (mean, min, max)
- `get_continuous`: High-frequency continuous (instantaneous) values
- `get_field_measurements`: Discrete measurements from field visits
- `get_monitoring_locations`: Site information and metadata
- `get_time_series_metadata`: A location's available data parameters
- `get_latest_daily`: Most recent daily statistical summary
- `get_latest_continuous`: Most recent high-frequency value
- `get_stats_por` / `get_stats_date_range`: Daily, monthly, and annual statistics
- `get_samples`: Discrete USGS water-quality samples

> - `get_ratings`: Stage-discharge rating curves (returns a `dict` of rating tables, not `(df, metadata)`)

Author: ```suggestion
- `get_ratings`: Stage-discharge rating curves
```
